truco-front/src/hooks/usePusherListeners.ts

```diff
@@ -153,10 +155,12 @@ export const usePusherListeners = (
         })
 
         gameManager.events.addOnFriendRequestListener((friendUser: User) => {
+            toast(friendUser.username + " sent you a friend request!")
             setFriendRequests((prev) => [...prev, friendUser])
         })
 
         gameManager.events.addOnGameChallengeListener((challenger: string) => {
+            toast(challenger + " challenged you to a game!")
             gameManager.acceptChallenge(challenger) // TODO replace with logic

```

Comment:
Maybe you want to do it on a later update, but here we need a way to accept or deny the challenge

Reply:
I think it's way outside the scope of this issue